manifests/resource/snippet.pp

```diff
@@ -0,0 +1,40 @@
+# define: nginx::resource::snippet
```

Comment:
Rework the documentation comments to be `puppet-strings` compatible/compliant.

Reply:
updated it.